app/javascript/components/recordings/RoomRecordings.jsx

```diff
@@ -11,10 +12,17 @@ import RoomsRecordingRow from './RoomsRecordingRow';
 
 export default function RoomRecordings() {
   const [input, setInput] = useState('');
+  const [page, setPage] = useState();
   const { friendlyId } = useParams();
-  const roomRecordings = useRoomRecordings(friendlyId, input);
+  const { isLoading, data: roomRecordings } = useRoomRecordings(friendlyId, input, page);
```

Comment:
Try `  const { isLoading, data: { data: recordings, meta: recordingsMeta } } = useRoomRecordings(friendlyId, input, page);`

Reply:
I tried this before. The issue with it is that the first `data` is still null when it tries to de-structure it and access the inner `data`, resulting in a `Cannot read properties of undefined (reading 'data')` error.

That's why I did it the way I did right now since it allows us to use the `?.` safe negation operator before trying to access the inner `data`.

I agree though that there should be a way to do what we are trying to do